cypress/support/login.ts

```diff
@@ -35,6 +35,15 @@ Cypress.Commands.add(
       cy.clearCookie('openshift-session-token');
 
       const idp = provider || KUBEADMIN_IDP;
+      // Check if OSD_ENV is defined.
+      const isOSD = Cypress.env('OSD_ENV') === 'y';
```

Comment:
I've kept `OSD_ENV` and `OSD_USERNAME` separate, but the latter makes the former redundant. However, I kept them this way for the sake of better readability, LMK if this looks odd and I'll remove it.

Reply:
Instead of creating 2 new variables, why not setting/checking `BRIDGE_HTPASSWD_USERNAME` ?
https://github.com/red-hat-storage/mcg-ms-console/blob/master/cypress/plugin.js#L74